wire-ios-sync-engine/Tests/Source/SessionManager/SessionManagerTests_CryptoboxMigration.swift

```diff
@@ -20,49 +20,49 @@ import XCTest
-    }
-
-    override func tearDown() {
-        mockCryptoboxMigrationManager = nil
-        proteusViaCoreCryptoFlag.isOn = false
-        super.tearDown()
-    }
-
-    func testItPerformsMigrationIfNeeded() {
-        // Given
-        mockCryptoboxMigrationManager.isMigrationNeededAccountDirectory_MockValue = true
-
-        // When
-        XCTAssert(login())
-
-        // Then
-        XCTAssertEqual(mockCryptoboxMigrationManager.performMigrationAccountDirectorySyncContext_Invocations.count, 1)
-        XCTAssertEqual(mockCryptoboxMigrationManager.completeMigrationSyncContext_Invocations.count, 1)
-    }
-
-    func testItDoesNotPerformMigration() {
-        // Given
-        mockCryptoboxMigrationManager.isMigrationNeededAccountDirectory_MockValue = false
-
-        // When
-        XCTAssert(login())
-
-        // Then
-        XCTAssertTrue(mockCryptoboxMigrationManager.performMigrationAccountDirectorySyncContext_Invocations.isEmpty)
-        XCTAssertEqual(mockCryptoboxMigrationManager.completeMigrationSyncContext_Invocations.count, 1)
-    }
-
-}
+// class SessionManagerTests_CryptoboxMigration: IntegrationTest {
+//
+//    var proteusViaCoreCryptoFlag = DeveloperFlag.proteusViaCoreCrypto
+//    var mockCryptoboxMigrationManager: MockCryptoboxMigrationManagerInterface!
+//
+//    override func setUp() {
+//        super.setUp()
+//        mockCryptoboxMigrationManager = MockCryptoboxMigrationManagerInterface()
+//        createSelfUserAndConversation()
+//
+//        mockCryptoboxMigrationManager.performMigrationAccountDirectorySyncContext_MockMethod = { _, _ in }
+//        mockCryptoboxMigrationManager.completeMigrationSyncContext_MockMethod = { _ in }
+////        sessionManager?.cryptoboxMigrationManager = mockCryptoboxMigrationManager
+//    }
+//
+//    override func tearDown() {
+//        mockCryptoboxMigrationManager = nil
+//        proteusViaCoreCryptoFlag.isOn = false
+//        super.tearDown()
+//    }
+//
+//    func testItPerformsMigrationIfNeeded() {
+//        // Given
+//        mockCryptoboxMigrationManager.isMigrationNeededAccountDirectory_MockValue = true
+//
+//        // When
+//        XCTAssert(login())
+//
+//        // Then
+//        XCTAssertEqual(mockCryptoboxMigrationManager.performMigrationAccountDirectorySyncContext_Invocations.count, 1)
+//        XCTAssertEqual(mockCryptoboxMigrationManager.completeMigrationSyncContext_Invocations.count, 1)
+//    }
+//
+//    func testItDoesNotPerformMigration() {
+//        // Given
+//        mockCryptoboxMigrationManager.isMigrationNeededAccountDirectory_MockValue = false
+//
+//        // When
+//        XCTAssert(login())
+//
+//        // Then
+//        XCTAssertTrue(mockCryptoboxMigrationManager.performMigrationAccountDirectorySyncContext_Invocations.isEmpty)
+//        XCTAssertEqual(mockCryptoboxMigrationManager.completeMigrationSyncContext_Invocations.count, 1)
+//    }
+//
+// }
```

Comment:
@typfel What's the reason for commenting out those tests? If we don't need them anymore, we can just delete them.

If there is a reason for them to be disabled because they're flaky, we can disable them through Xcode Interface and create a ticket to keep track of it and come back at them at a later time

Reply:
Yeah, I forgot delete them

Reply:
And gone 💥